src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -341,16 +348,26 @@ protected WebResourceResponse interceptRequest(String rawUrl) {
     return res;
   }
 
+  private void callJavaScriptFunction(String func) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+      if (internetAccess) {
+        webView.evaluateJavascript("window." + func + ";", null);
```

Comment:
@r10s with this we should avoid any possible issue of "URL limit" of the old "javascript:" URL hack